Task_TheGridProblem/index.css

```diff
@@ -1 +1,79 @@
-/* Start coding below for GRID PROBLEM TASK */
+.container{
+  display:grid;
+  grid-template-rows: 200px 200px 200px;
+  grid-template-columns: 200px 200px 200px;
+  background-color: #a2d2ff;
+  padding: 30px;
```

Comment:
The use of 'gap' is not supported in all browsers. It is recommended to use 'grid-gap' instead.